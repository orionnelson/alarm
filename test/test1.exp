alarm> Main Thread Received Alarm Request Number: (1) Alarm Request: <Good Morning> 
alarm> Alarm Thread Passed on Alarm Request to Display Thread 1 Alarm Request Number: (1) Alarm Request: Good Morning 
Display Thread 1: Recieved Alarm Request Number:(1) Alarm Request: <Good Morning> 
Display Thread 1: Number of SecondsLeft <5>: Alarm Request Number: (1) Alarm Request: <Good Morning> 
Display Thread 1: Number of SecondsLeft <3>: Alarm Request Number: (1) Alarm Request: <Good Morning> 
Display Thread 1: Number of SecondsLeft <1>: Alarm Request Number: (1) Alarm Request: <Good Morning> 
Display Thread 1: Alarm Expired at 1637029413: Alarm Request Number: (1) Alarm Request Good Morning 
