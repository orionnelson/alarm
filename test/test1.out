alarm> Main Thread Received Alarm Request Number: (1) Alarm Request: <Good Morning> 
alarm> 